client/app/core/event-notifications.service.js

```diff
@@ -154,26 +155,30 @@ export function EventNotificationsFactory ($timeout, lodash, CollectionsApi, Ses
     removeToast(notification)
     service.markRead(notification)
   }
+
   function setToastDisplay (enabled) {
     state.toastsEnabled = enabled
   }
+
   // Private
   function add (notificationType, type, message, notificationData, id) {
-    const group = lodash.find(state.groups, {notificationType: notificationType})
-    const newNotification = {
-      id: id,
-      notificationType: notificationType,
-      unread: angular.isDefined(notificationData) ? notificationData.unread : true,
-      type: type,
-      message: message,
-      data: notificationData || {},
-      href: id ? '/api/notifications/' + id : undefined,
-      timeStamp: (new Date()).getTime()
-    }
+    if (!RBAC.has(RBAC.FEATURES.CORE.NOTIFICATIONS)) {
```

Comment:
This should probably not be negative...

`RBAC.has('sui_notifications')` gives me `true`... but no notifications ever appear :).